### PR TITLE
Landing page

### LINKED ISSUE #15: [WIDOK] Landing page

### PR DESCRIPTION
**Pull request by wzieba**
Header and footer partials

Closes #15

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -4,12 +4,18 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
 
+
   <body>
+  
+<%= render "shared/header" %>
```
Comment (wzieba) [inline app/views/layouts/application.html.erb]: Please align line

File: app/views/shared/_footer.html.erb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<footer class="footer">
+      <div class="container">
+        <span class="text-muted">Copywright by Awsome Netgutu Team 2</span>
+      </div>
+    </footer>
```
Comment (wzieba) [inline app/views/shared/_footer.html.erb]: Remove tabulation here please

File: app/views/shared/_header.html.erb [hunk 3]
```diff
@@ -0,0 +1,35 @@
+<nav class="navbar navbar-expand-lg navbar-light bg-light">
+    <a class="navbar-brand" href="#">
+    
+<%= image_tag "map-location.svg", alt: "Logo",
```
Comment (wzieba) [inline app/views/shared/_header.html.erb]: Please move this like `<a(...)> content </a>`

File: test/controllers/welcome_controller_test.rb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
```
Comment (wzieba) [inline test/controllers/welcome_controller_test.rb]: Please remove this test

File: app/views/shared/_footer.html.erb (hunk 2 above)
Comment (dmp0x7c5) [reply app/views/shared/_footer.html.erb]: *indentation :)

**Pull request merged**